UnityProject/Assets/Scripts/HealthV2/Living/BodyPartFunctionality.cs

```diff
@@ -12,13 +13,20 @@ public class BodyPartFunctionality : NetworkBehaviour, IEmpAble
 
 
 		public virtual void ImplantPeriodicUpdate(){}
-		public virtual void OnRemovedFromBody(LivingHealthMasterBase livingHealth){}
+
+		public virtual void OnRemovedFromBody(LivingHealthMasterBase livingHealth)
```

Comment:
Wouldn't this man the logs since this is for everybody part * body part functions?

Reply:
updated